assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js

```diff
@@ -87,6 +95,32 @@ export default function ReminderBanner( {
 		ACTIVATION_ACKNOWLEDGEMENT_TOOLTIP_STATE_KEY
 	);
 
+	const viewContext = useViewContext();
+	const eventCategory = `${ viewContext }_ga4-reminder-notification`;
+
+	useMount( () => {
+		if (
+			! isTooltipVisible &&
+			( ! isLoadingAnalyticsAccess || isDismissed )
```

Comment:
I think this condition should be tweaked because we _don't_ want to track the `view_notification` event when `isDismissed` is true, because in that case the `BannerNotification` itself will return `null`, meaning the notification is not shown.
```suggestion
			! ( isLoadingAnalyticsAccess || isDismissed )
```

Reply:
Ah makes sense. I just inversed the conditional that shows the `ProgressBar` and came to that expression. Thanks!